### PR TITLE
Logic that trains model on the kaggle data

### PR DESCRIPTION
**Pull request by b0noI**

### CODE REVIEW

File: src/main/java/com/dj/models/mnist/MnistTrainer.java [hunk 1]
```diff
@@ -27,17 +34,15 @@
 
 public class MnistTrainer {
 
-    public static void trainMnistNN(final boolean debug) {
+    private static final int INDEX_OF_KAGGLE_IMAGES = 0;
+    private static final int INDEX_OF_KAGGLE_LABELS = 1;
```
Comment (andrey-kotkovets) [inline src/main/java/com/dj/models/mnist/MnistTrainer.java]: @b0noI Add an empty line between 2 constants.

File: src/main/java/com/dj/models/mnist/MnistTrainer.java [hunk 2]
```diff
@@ -27,17 +34,15 @@
...
+
+    public static void downloadDataAndtrainMnistNN(final boolean debug) {
         final ModelWrapper modelWrapper = createTheModel(debug);
-        trainMnistNN(modelWrapper);
+        downloadDataAndtrainMnistNN(modelWrapper);
```
Comment (andrey-kotkovets) [inline src/main/java/com/dj/models/mnist/MnistTrainer.java]: @b0noI ...AndTrain... - T is a capital letter.

File: src/main/java/com/dj/models/mnist/MnistTrainer.java (hunk 1 above)
Comment (b0noI) [reply src/main/java/com/dj/models/mnist/MnistTrainer.java]: done

File: src/main/java/com/dj/models/mnist/MnistTrainer.java (hunk 2 above)
Comment (b0noI) [reply src/main/java/com/dj/models/mnist/MnistTrainer.java]: done

**Pull request merged**